src/geant4/app/src/ExN01UserScoreWriter.cc

```diff
@@ -30,7 +30,11 @@ void ExN01UserScoreWriter::DumpAllQuantitiesToFile(const G4String& fileName,
   G4cout << "Dumping mesh " << fScoringMesh->GetWorldName() << " to file"
          << G4endl;
 
-  // retrieve the map
+// retrieve the map
+#ifdef GEANT4_GT_10_6
+  using MeshScoreMap = G4VScoringMesh::MeshScoreMap;
+#endif
+
```

Comment:
wasn't this already in an earlier PR?

Reply:
As I recall, this should be required for >10.6 and It may need to be available in earlier PR.